### DISCUSSION TITLE
Updating takes up to 10 minutes

### DISCUSSION THREAD

**Discussion by JoshClose** [containrrr/watchtower#1743] [General]
After I push a new image labeled `test`, it takes up to 10 minutes for Watchtower to update it. I have Watchtower running every 10 seconds. My container registry is in DigitalOcean, and they said there is a tag TTL of 10 minutes, which is causing this.

Is there a way to have Watchtower ignore this tag TTL when pulling?

**Comment by piksel** [comment]
I think the TTL just means that the image won't change more often than that. Watchtower doesn't know about any TTL, it just asks what the latest version of the image is.
